essentials.yml

```diff
@@ -0,0 +1,4 @@
+---
+ath:
+  useLocalSnapshots: false
```

Comment:
I would think this should be the default. Really you should never use `LOCAL_SNAPSHOTS` to begin with—only `LOCAL_JARS`, which should be implied by other configuration where applicable.

Reply:
Internally runATH uses `LOCAL_JARS` but I believe `useLocalSnapshots` expresses better the meaning of this parameter (test using a locally generated snapshot of a plugin) for non ATH savvy people. IMO `LOCAL_JARS` and `LOCAL_SNAPSHOTS`